packages/file_selector/file_selector_platform_interface/lib/src/types/x_type_group/x_type_group.dart

```diff
@@ -14,7 +15,22 @@ class XTypeGroup {
     this.mimeTypes,
     this.macUTIs,
     this.webWildCards,
-  });
+  }) {
+    _verifyExtensions();
+  }
+
+  void _verifyExtensions() {
+    if (extensions == null) return;
+    for (var i = 0; i < extensions!.length; i++) {
+      if (!extensions![i].startsWith('.')) continue;
+      if (kDebugMode) {
+        print('extensions[${i}] with value "${extensions![i]}" is invalid.'
+            ' We are mutating the extensions list to remove the leading dot.'
```

Comment:
"We" is very ambiguous here. How about "The leading dots are being removed from the extensions".

Reply:
Agree. Done. PTAL